scripts/dbsetup.sql

```diff
@@ -68,3 +76,10 @@ CREATE TRIGGER set_timestamp_healthit_products
 BEFORE UPDATE ON healthit_products
 FOR EACH ROW
 EXECUTE PROCEDURE trigger_set_timestamp();
+
+
+CREATE or REPLACE VIEW org_mapping AS
+SELECT endpts.url, orgs.name, orgs.Location->>'state'
```

Comment:
```suggestion
SELECT endpts.url, orgs.name, orgs.Location->>'state' AS state
```

this provides a column name

Reply:
include the endpoint's organization name and the organization's secondary name.